src/Libraries/RevitNodes/Elements/InternalUtilities/ElementUtils.cs

```diff
@@ -187,38 +194,142 @@ public static void SetParameterValue(Autodesk.Revit.DB.Parameter param, bool val
         public static double GetConvertedParameterValue(Autodesk.Revit.DB.Parameter param, double value)
         {
             var paramType = param.Definition.ParameterType;
```

Comment:
I did a test in a python node, and this seems to work:
![image](https://cloud.githubusercontent.com/assets/5968613/22456987/cac2d578-e795-11e6-8737-527abf703536.png)
```
import clr

clr.AddReference('RevitAPI')
from Autodesk.Revit.DB import *

clr.AddReference('RevitNodes')
import Revit
clr.ImportExtensions(Revit.GeometryConversion)
clr.ImportExtensions(Revit.Elements)

clr.AddReference('RevitServices')
import RevitServices
from RevitServices.Persistence import DocumentManager
from RevitServices.Transactions import TransactionManager

doc = DocumentManager.Instance.CurrentDBDocument
uidoc=DocumentManager.Instance.CurrentUIApplication.ActiveUIDocument

# Get parameter and unit type from element
element = UnwrapElement(IN[0])
bip = BuiltInParameter.WALL_BASE_OFFSET
parameter = element.get_Parameter(bip)
unitType = parameter.Definition.UnitType

# Get the display unit set for the unit type in the current document
formatOption = doc.GetUnits().GetFormatOptions(unitType)
displayUnit = formatOption.DisplayUnits

#Convert the value with UnitUtils
OUT = UnitUtils.ConvertFromInternalUnits(parameter.AsDouble(),displayUnit)
```